Avoid 64b typing error on RPC with 64b members in structures, while we migrate
to the fixed 64b structure alignment.

To be removed after people have migrated both gnumach libc+hurd rebuilt against
fixed mig.

Index: glibc-2.41/mach/Makefile
===================================================================
--- glibc-2.41.orig/mach/Makefile
+++ glibc-2.41/mach/Makefile
@@ -63,6 +63,9 @@ CFLAGS-RPC_i386_set_gdt.o = $(no-stack-p
 CFLAGS-RPC_i386_set_ldt.o = $(no-stack-protector)
 CFLAGS-RPC_task_get_special_port.o = $(no-stack-protector)
 CFLAGS-RPC_thread_set_state.o = $(no-stack-protector)
+CFLAGS-RPC_host_get_time.c = -DTypeCheck=0
+CFLAGS-RPC_host_get_time64.c = -DTypeCheck=0
+CFLAGS-RPC_host_get_uptime64.c = -DTypeCheck=0
 
 # Translate GNU names for CPUs into the names used in Mach header files.
 mach-machine = $(patsubst powerpc,ppc,$(base-machine))
Index: glibc-2.41/hurd/Makefile
===================================================================
--- glibc-2.41.orig/hurd/Makefile
+++ glibc-2.41/hurd/Makefile
@@ -182,6 +182,20 @@ generated += $(inlines:=.c)
 CFLAGS-hurdstartup.o = $(no-stack-protector)
 CFLAGS-RPC_exec_startup_get_info.o = $(no-stack-protector)
 
+CFLAGS-RPC_file_record_lock.c = -DTypeCheck=0
+CFLAGS-RPC_file_statfs.c = -DTypeCheck=0
+CFLAGS-RPC_file_utimens.c = -DTypeCheck=0
+CFLAGS-RPC_file_utimes.c = -DTypeCheck=0
+CFLAGS-RPC_io_select_timeout.c = -DTypeCheck=0
+CFLAGS-RPC_io_select_timeout_request.c = -DTypeCheck=0
+CFLAGS-RPC_io_select_timeout_request.os= -DTypeCheck=0
+CFLAGS-RPC_io_stat.c = -DTypeCheck=0
+CFLAGS-RPC_io_stat_reply.c = -DTypeCheck=0
+CFLAGS-RPC_login_get_idle_time.c = -DTypeCheck=0
+CFLAGS-RPC_proc_getchildren_rusage.c = -DTypeCheck=0
+CFLAGS-RPC_proc_wait.c = -DTypeCheck=0
+CFLAGS-RPC_proc_waitid.c = -DTypeCheck=0
+
 # Make it simpler to unwind over INTR_MSG_TRAP
 CFLAGS-intr-msg.o = -fno-omit-frame-pointer
 CFLAGS-intr-msg.os = -fno-omit-frame-pointer
